blocks/blockstore/write_cache.go

```diff
@@ -22,6 +22,7 @@ type writecache struct {
 }
 
 func (w *writecache) DeleteBlock(k key.Key) error {
+	defer log.EventBegin(context.TODO(), "Removed block ", &k).Done()
```

Comment:
IIRC, the trailing space was needed to separate the key and "Removed block" in the text logs (not json).